src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
```

Comment:
Indeed you are correct. I hadn't thought that through... It would automatically bundle all "elements". We're targeting small customer builds as part of our goals, #524. So from now on let's stick with directly importing files. Thanks for pointing it out.